arch/riscv/Kconfig.core

```
@@ -0,0 +1,18 @@
+# Copyright (c) 2022 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menu "RISCV core"
+
+config RISCV_CORE_E31
+	bool "Use E31 core"
```

`Use`, as with `enable`, is somewhat redundant.

```suggestion
	bool "E31 core"
```

Also, this should be promptless.